tests/integration/test_dummy.py

```diff
@@ -1,2 +1,15 @@
-def test_dummy_unit() -> None:
-    assert 1 == 1  # noqa: S101
+from adcm_aio_client.core.objects.cm import ClustersNode
+from adcm_aio_client.core.requesters import DefaultRequester
+from adcm_aio_client.core.types import Credentials
+from tests.integration.setup_environment import SetupManager, adcm_image_name, postgres_image_name
+
+
+async def test_clusters_page() -> None:
+    with SetupManager(postgres_image_name, adcm_image_name) as container:
+        requester = DefaultRequester(base_url=container.adcm_container.url, retries=3, retry_interval=15, timeout=30)
+        accessor = ClustersNode(requester=requester, path=("clusters",))
+
+        await requester.login(Credentials(username="admin", password="admin"))  # noqa: S106
+        clusters = await accessor.list()
+
```

Comment:
let's improve this example:

Instead of building node manually, an instance of ADCM client should be used

So let's define fixture `client` in here that will prepare and return client.
This fixture will be dependant on `adcm` fixture mentioned in previous comments.

Test itself should use this client, making request (as it is now).
And for demonstrating purposes let's check that connection is truly to postgresql (either checking for tables in there or absence of sqlite db file in adcm container)

Reply:
reworked with fixtures

Reply:
> So let's define fixture client in here that will prepare and return client.